testsuite/cluster-test/src/deployment.rs

```diff
@@ -19,7 +19,9 @@ pub struct DeploymentManager {
 }
 
 const LAST_DEPLOYED_FILE: &str = ".last_deployed_digest";
-const REPOSITORY_NAME: &str = "libra_e2e";
+const VALIDATOR_IMAGE_NAME: &str = "libra_e2e";
```

Comment:
Strictly speaking this is repository name, not image name

Reply:
Updated to call it `VALIDATOR_IMAGE_REPO`